apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx

```diff
@@ -155,6 +161,8 @@ export function LogDrainDestinationSheetForm({
       url: defaultValues?.config?.url || '',
       api_key: defaultValues?.config?.api_key || '',
       region: defaultValues?.config?.region || '',
+      header1_key: defaultValues?.config?.header1_key || '',
```

Comment:
Should use the same header logic as http destination, can be one or more headers with the application/json header set by default

Reply:
Should be the `headers` key